Packs/CommunityCommonScripts/Scripts/BulkCloseIncidents/BulkCloseIncidents.py

```diff
@@ -0,0 +1,54 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+def send_post_request(endpoint, data=None):
+    try:
+        if data:
+            data = json.dumps(data)
+            res = execute_command("core-api-post", {
+                "uri": endpoint,
+                "body": data
+            })
+    except requests.HTTPError as e:
+        return_error(f"HTTP error: {e}")
+    return res
+
+
+def main():
+    args = demisto.args()
+    query = args.get('query')
+    try:
+        batch_size = int(args.get('batch_size'))
+    except ValueError:
+        return_error("'batch_size' must be integer.")
+    try:
+        # only get active incidents (status == 1)
+        incidents = [inc['id'] for inc in execute_command('getIncidents', {
```

Comment:
Direct dictionary access is used to retrieve 'data', 'id', and 'status' (line 29). This can cause a `KeyError` if the API response does not contain these keys. Please use the `.get()` method for safer access.